SofascoreAcademy/SofascoreAcademy/View/MatchView.swift

```diff
@@ -0,0 +1,210 @@
+        time.text = dateFormatter.string(from: matchModel.startTime)
+        time.font = Fonts.regularCondensedFont.withSize(Fonts.smallSize)
+        time.textColor = Colors.onSurfaceLv2
+        time.textAlignment = .center
+        
+        divider.backgroundColor = Colors.onSurfaceLv4
+        
+        team1Logo.image = matchModel.team1.logo
+        team2Logo.image = matchModel.team2.logo
+        
+        minutes.font = Fonts.regularCondensedFont.withSize(Fonts.smallSize)
+        minutes.textColor = Colors.onSurfaceLv2
+        minutes.textAlignment = .center
+        
+        team1Name.text = matchModel.team1.name
+        team1Name.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team1Name.textColor = Colors.onSurfaceLv1
+        
+        team2Name.text = matchModel.team2.name
+        team2Name.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team2Name.textColor = Colors.onSurfaceLv1
+        
+        team1Score.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team1Score.textColor = Colors.onSurfaceLv1
+        team1Score.textAlignment = .right
+        
+        team2Score.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team2Score.textColor = Colors.onSurfaceLv1
+        team2Score.textAlignment = .right
+        switch matchModel.status {
+        case .finished:
+            minutes.text = Strings.finishedMatch
+            
+            guard let score1 = matchModel.team1Score else {
+                return
+            }
+            guard let score2 = matchModel.team2Score else {
+                return
+            }
+            
+            team1Score.text = String(score1)
+            team2Score.text = String(score2)
+            
+            if score1 < score2 || score1 == score2 {
+                team1Name.textColor = Colors.onSurfaceLv2
+                team1Score.textColor = Colors.onSurfaceLv2
+            }
+            if score1 > score2 || score1 == score2 {
+                team2Name.textColor = Colors.onSurfaceLv2
+                team2Score.textColor = Colors.onSurfaceLv2
+            }
+        case .inProgress:
+            guard let minute = matchModel.currentMinute else {
+                return
+            }
+            minutes.text = String(minute) + "'"
+            minutes.textColor = Colors.live
+            
+            guard let score1 = matchModel.team1Score else {
+                return
+            }
+            guard let score2 = matchModel.team2Score else {
+                return
+            }
+            
+            team1Score.text = String(score1)
+            team2Score.text = String(score2)
+            
+            team1Score.textColor = Colors.live
+            team2Score.textColor = Colors.live
+        case .notStarted:
+            minutes.text = "-"
+        }
+    }
+    
+    override func setupConstraints() {
+        self.snp.makeConstraints() {
+            $0.height.equalTo(56)
+        }
```

Comment:
Ovdje ti isto ne treba visina.
I koliko vidim xcode se zali na constraintove. Probaj vidjeti što ti fali.

![image](https://github.com/ana-bagic/Sofascore2024/assets/8843138/a2e7d9b3-4d0e-4997-ade2-723c8786290f)

Mislim da isto negdje fali da nisi bottom definirala. Ugl praksa je inače da dobro bottom i top definiraš i onda ti ne treba hight nego content unutar viewa definira height